components/tokens/custom-spectrum/custom-vars.css

```diff
@@ -35,16 +35,13 @@ governing permissions and limitations under the License.
   --spectrum-animation-ease-out: cubic-bezier(0, 0, 0.40, 1);
   --spectrum-animation-ease-linear: cubic-bezier(0, 0, 1, 1);
 
-  --spectrum-sans-serif-font: var(--spectrum-sans-serif-font-family);
-  --spectrum-sans-font-family-stack: var(--spectrum-sans-serif-font), adobe-clean, 'Source Sans Pro', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Ubuntu, 'Trebuchet MS', 'Lucida Grande', sans-serif;
+  --spectrum-sans-font-family-stack: adobe-clean, 'Source Sans Pro', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Ubuntu, 'Trebuchet MS', 'Lucida Grande', sans-serif;
```

Comment:
Should we leave the Adobe Clean in the mix? If we add it second, then at least the (probably internal) user would fallback to the local font instead of a different set entirely. Or is that too confusing for folks? It would only be used if the web font is not loaded.

Reply:
Updated. Let me know what you think of this most recent commit.